src/components/Shifts/AddShiftForm.tsx

```diff
@@ -25,13 +25,21 @@ export interface ShiftsFormProps {
 
 export const AddShiftForm = ({port, terminal, interval, shiftForms, confirmHandler}: ShiftsFormProps) => {
   const [shifts, setShifts] = useState<ShiftForm[]>(Array.from(shiftForms).length > 0 ? shiftForms : [
-    {id: 1, name: 'Shift 1', startTime: '00:00', endTime: '01:00', defaultStaffNumber: 0}
+    {id: 1, name: '', startTime: '', endTime: '', defaultStaffNumber: 0}
   ]);
   const [showConfirm, setShowConfirm] = useState(false);
   const [error, setError] = useState(false);
+  const isFirstRender = useRef(true);
```

Comment:
What's the purpose of `isFirstRender`? Feels like there's probably a more intuitive way of handling whatever it's doing

Reply:
This is when form is new shift form is created not to worry about errors